libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/ExecSpecRule/Conway/Ledger.hs

```diff
@@ -151,6 +155,24 @@ instance
       . computationResultToEither
       $ Agda.ledgerStep env st sig
 
+  extraInfo
+    globals
+    ConwayLedgerExecContext {..}
+    LedgerEnv {..}
+    LedgerState {..}
+    sig
+    _ =
+      extraInfo @fn @"UTXOW" @Conway
+        globals
+        clecUtxoExecContext
+        utxoEnv
+        lsUTxOState
+        sig
+        stFinal
+      where
+        utxoEnv = UtxoEnv ledgerSlotNo ledgerPp lsCertState
+        stFinal = runSTS @"UTXOW" @Conway globals utxoEnv lsUTxOState sig
```

Comment:
How come you chose UTXOW to add to extra info? Is it the most likely to have relevant information if the test fails?

Reply:
Wait I'm confused, how is this definition related to the `extraInfo` invocation below ? 
```
let extra = extraInfo @fn @"LEDGER" @Conway globals ctx (inject env) (inject st) (inject sig) implRes
```

Reply:
I also see that in the extraInfo for UTXOW you're calling UTXO  - so I guess it's a way to see some "partial results" of the rule invocation?

Reply:
Yeah, this way I can still get some debug information if I only call the `UTXOW` rule. When I call `UTXO`, I get all of the information in `UTXOW`, plus any additional info that I can provide with the additional information available in `UTXO`.